### PR TITLE
Fix Broken Link

### LINKED ISSUE #634: [Bug]: Broken Link
### What happened?

The link given at https://www.jaegertracing.io/docs/1.45/#related-links  for "IBM Garage Cloud Native Bootcamp - Distributed Tracing(tutorials)" is broken

### Steps to reproduce

1. Go to https://www.jaegertracing.io/docs/1.45/#related-links 
2. Click on the "IBM Garage Cloud Native Bootcamp - Distributed Tracing(tutorials)" link

### Expected behavior

Link Should Work

### Screenshot

image
image

### PR DESCRIPTION
**Pull request by SaarthakMaini**
## Which problem is this PR solving?
- Resolves #634

## Short description of the changes
- Fixed Broken Link

### CODE REVIEW

File: content/docs/1.45/_index.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ See [running a docker all in one image](getting-started#all-in-one).
 - [Evolving Distributed tracing At Uber Engineering](https://eng.uber.com/distributed-tracing/) (blog)
 - [Mastering Distributed Tracing](https://shkuro.com/books/2019-mastering-distributed-tracing/) (book)
 - [OpenTracing Tutorial (Java, Go, Python, Node.js, C#)](https://github.com/yurishkuro/opentracing-tutorial/) (tutorials)
-- [IBM Garage Cloud Native Bootcamp - Distributed Tracing](https://cloudnative101.dev/electives/dist-trace/) (tutorials)
+- [IBM Garage Cloud Native Bootcamp - Distributed Tracing](https://www.ibm.com/cloud/blog/distributed-tracing) (tutorials)
```
Comment (yurishkuro) [inline content/docs/1.45/_index.md:72]: From what I remember of the original link, this is not an equivalent replacement. The original was an actual course, where this blog post is mostly fluff. If we cannot find the original course I suggest we simply remove this entry.

**Review by yurishkuro — CHANGES_REQUESTED** [review]

File: content/docs/1.45/_index.md (hunk 1 above)
Comment (yurishkuro) [reply content/docs/1.45/_index.md:72]: Also, we should make the same change across all versioned docs, not just the latest version.

Comment (SaarthakMaini) [reply content/docs/1.45/_index.md:72]: I understand. Thank you very much for the review, I'll make the changes soon

Comment (SaarthakMaini) [reply content/docs/1.45/_index.md:72]: I'm not able to find the original course. The links which do look similar are pointing to cloudnative101.dev which is not working.
So I think the best course of action would be to remove the links

**Comment by SaarthakMaini** [comment]
The necessary changes have been made. 

Please feel free to share any feedback/improvements. 

Thank you!

**Review by yurishkuro — APPROVED** [review]
Thanks!

**Pull request merged**